src/main/java/com/minecolonies/coremod/colony/buildings/AbstractSchematicProvider.java

```diff
@@ -306,53 +290,26 @@ public boolean hasParent()
     @Override
     public void setParent(final BlockPos pos)
     {
-        if (!pos.equals(getID()))
```

Comment:
Not sure about this part, wont this maybe override a valid parent in the worst case?

Reply:
yeah I know, but after it derped the parent up, a potentially valid parent was replaced